gen/generator.go

```diff
@@ -849,7 +863,8 @@ func genAnnotationUnsupported() []string {
 }
 
 func main() {
-	getClassMetadata()
+	reGenClassMetadata()
```

Comment:
Shall we include the full name? 
reGenerateClassMetadata()

Reply:
Done